tick/base/base.py

```diff
@@ -547,4 +555,8 @@ def _inc_attr(self, key: str, step: int = 1):
         self._set(key, getattr(self, key) + step)
 
     def __str__(self):
-        return json.dumps(self._as_dict(), sort_keys=True, indent=2)
+
+        dic = self._as_dict()
+        if 'dtype' in dic:
+            del dic['dtype']
+        return json.dumps(dic, sort_keys=True, indent=2)
```

Comment:
Why should we delete dtype from dic?

Reply:
It's not serializable

Reply:
yep seems to work now as everything is a string

Reply:
Actually fails in doc stuff, so I'll leave it